services/discordService.js

```diff
@@ -51,8 +51,25 @@ const addRoleToUser = async (userid, roleid) => {
   return response;
 };
 
+const removeRoleFromUser = async (roleId, discordId) => {
+  try {
+    const authToken = await generateAuthTokenForCloudflare();
+    const data = await fetch(`${DISCORD_BASE_URL}/roles`, {
+      method: "DELETE",
+      headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+      body: JSON.stringify({ userid: discordId, roleid: roleId }),
```

Comment:
I suggest we should update the API to expect these IDs as `userId` and `roleId`

Reply:
We cant do that because the discord uses userid and roleid so that requires a long chain of change